domain/src/main/scala/com/all4journey/domain/Tables.scala

```diff
@@ -16,7 +14,7 @@ trait Tables {
   import slick.jdbc.{GetResult => GR}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/rjkernick/all4journey/file/2445603256/issues/source?bid=3246460&fileBranchId=3246461#l14)